.github/ISSUE_TEMPLATE/bug_report_form.md

```diff
@@ -0,0 +1,87 @@
+name: Bug report form for non-technical users
```

Comment:
I think we should avoid describing this as being for "non-technical" users since that could come across as demeaning. How about we just frame it as an alternative that anyone can use if they'd prefer?

Reply:
Simplified Bug report form?

Reply:
Sounds good!

Reply:
Done